### PR TITLE
Add the clevis playbook

### PR DESCRIPTION
**Pull request by matfechner**
This playbook will install the clevis role which is required
by network bound disk encryption as client part

Signed-off-by: Mathias Fechner <fechner@osism.tech>

### CODE REVIEW

File: playbooks/infrastructure/clevis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+- name: Apply role clevis
+  hosts: "{{ hosts_clevis|default('clevis') }}"
+  serial: "{{ osism_serial['clevis']|default(osism_serial_default)|default(0) }}"
+
+  collections:
+    - osism.services
+    - community.crypto
```
Comment (berendt) [inline playbooks/infrastructure/clevis.yml:8]: Why is this required here?

Comment (matfechner) [reply playbooks/infrastructure/clevis.yml:8]: @berendt  community.crypto is obselete i will fix it

**Comment by berendt** [comment]
Depends on https://github.com/osism/ansible-collection-commons/pull/343

File: playbooks/infrastructure/clevis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+- name: Apply role clevis
+  hosts: "{{ hosts_clevis|default('clevis') }}"
+  serial: "{{ osism_serial['clevis']|default(osism_serial_default)|default(0) }}"
+
+  collections:
+    - osism.services
```
Comment (berendt) [inline playbooks/infrastructure/clevis.yml:7]: This should be osism.commons and not osism.services.

**Review by berendt — CHANGES_REQUESTED** [review]

File: playbooks/infrastructure/clevis.yml (hunk 2 above)
Comment (matfechner) [reply playbooks/infrastructure/clevis.yml]: done
